profiler/profiler.go

```diff
@@ -70,6 +70,9 @@ func newProfiler(opts ...Option) (*profiler, error) {
 		opt(cfg)
 	}
 	if cfg.apiKey != "" {
+		if !isAPIKeyValid(cfg.apiKey) {
+			return nil, fmt.Errorf("API key has incorrect format")
```

Comment:
```suggestion
			return nil, errors.New("API key has incorrect format")
```
Small nit. No need to format here.

Reply:
I'll take it!